src/Module/Contact/Profile.php

```diff
@@ -24,6 +24,7 @@
 use Friendica\App;
 use Friendica\BaseModule;
 use Friendica\Contact\LocalRelationship;
+use Friendica\Contact\LocalRelationship\Entity\LocalRelationship as EntityLocalRelationship;
```

Comment:
Please rename the alias to `LocalRelationshipEntity` to satisfy the English order of qualifier.
```suggestion
use Friendica\Contact\LocalRelationship\Entity\LocalRelationship as LocalRelationshipEntity;
```